internal/dag/dag.go

```diff
@@ -429,6 +429,12 @@ type RetryPolicy struct {
 	// PerTryTimeout specifies the timeout per retry attempt.
 	// Ignored if RetryOn is blank.
 	PerTryTimeout timeout.Setting
+
+	// SkipPreviousHost configures Envoy to use Previous Host Retry Predicate
+	// that allows it to choose a different host than the host where previous request
+	// has failed.
+	// +optional
+	SkipPreviousHost bool `json:"SkipPreviousHost,omitempty"`
```

Comment:
Can remove the struct tag and the `+optional`

Reply:
fixed in 0641aa3d74a102e52039fa9a4f51a0349060eae5

Reply:
I actually reverted that change because we want the `+optional` such that the generated CRDs have this marked as optional

Reply:
otherwise all `httpproxies` that dont have this `flag` fail the CRD validation

Reply:
fixed